helm/scripts/shared-resources-postsync.sh

```
@@ -33,6 +48,9 @@ create_or_update_configmap shared-config \
   --from-literal=TEMPORAL_IP="$TEMPORAL_IP" \
   --from-literal=UMAMI_URL="$UMAMI_URL" \
   --from-literal=UMAMI_WEBSITE_NAME="$UMAMI_WEBSITE_NAME" \
+  --from-literal=admin_allowed_users="$admin_allowed_users"
```

Shellcheck (0.11.0)

[warning] admin_allowed_users is referenced but not assigned.

(SC2154)